src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -398,4 +411,89 @@ class TableView extends React.Component<TableViewProps> {
   }
 }
 
+const UNSEARCHABLE_FIELDS = ['p99', 'p75', 'p95', 'last_seen'];
+
+const ExpandAggregateRow = (props: {
+  children: React.ReactNode;
+  eventView: EventView;
+  column: TableColumn<keyof TableDataRow>;
+  dataRow: TableDataRow;
+  location: Location;
+}) => {
+  const {children, column, dataRow, eventView, location} = props;
+
+  const {eventViewField} = column;
+
+  const exploded = explodeField(eventViewField);
+  const {aggregation} = exploded;
+
+  if (aggregation === 'count') {
+    return (
+      <Tooltip title={t('Expand aggregated row')}>
+        <a
+          href="#expand"
+          onClick={event => {
+            event.preventDefault();
+
+            const nextEventView = eventView.clone();
+
+            const additionalSearchConditions: string[] = [];
+
+            nextEventView.fields = nextEventView.fields.map(
+              (field: Field): Field => {
+                if (eventViewField.field === field.field) {
+                  // convert all instances of count(exploded.field) to exploded.field
+                  return {
+                    field: exploded.field,
+                    // enforce the renamed title
+                    // e.g "# of events" to become "id"
+                    title: exploded.field,
+                  };
+                }
+
+                const currentExplodedField = explodeField(field);
+                if (currentExplodedField.aggregation) {
+                  // this is a column with an aggregation; we skip this
+                  return field;
+                }
+
+                if (UNSEARCHABLE_FIELDS.includes(currentExplodedField.field)) {
+                  return field;
+                }
+
+                // add this field to the search conditions
+
+                const dataKey = getAggregateAlias(field.field);
+                const value = dataRow[dataKey];
+
+                if (value) {
+                  additionalSearchConditions.push(
+                    `${currentExplodedField.field}:"${value}"`
+                  );
+                }
+
+                return field;
+              }
+            );
+
+            nextEventView.query = `${
+              nextEventView.query
+            } ${additionalSearchConditions.join(' ')}`.trim();
```

Comment:
Do you need to handle the same key appearing more than one time?

Reply:
Should be addressed in https://github.com/getsentry/sentry/pull/15793/commits/9af1a8aaf1c1b78738d12fef918c166c9a5b5c1e

I've made use of the utility functions to address this from this file: https://github.com/getsentry/sentry/blob/fd6e8be4e47fbab6a0d9f848af483d70ef56ddd9/src/sentry/static/sentry/app/utils/tokenizeSearch.tsx